libbf.c

```diff
@@ -179,10 +179,6 @@ static inline __maybe_unused limb_t shld(limb_t a1, limb_t a0, long shift)
         return a1;
 }
 
-#define malloc(s) malloc_is_forbidden(s)
```

Comment:
Thsi ones are here to prevent accidental allocations, they need to exist. Perhaps we could skip them with some define @chqrlie ?

Reply:
I think the easiest way to maintain current behaviour would be to just `#undef` them at the end of the respective source file, so that the (intentional) `malloc` calls in subsequent sources aren't affected in a jumbo build.

How does that sound?

Reply:
Sounds good!

Reply:
> Sounds good!

Great! Updated the changes, safe for re-review when you've got a minute

Reply:
Ah hold on, was just testing with cmake's `-DCMAKE_UNITY_BUILD=ON` and there are a few more collisions

Reply:
That should be all for now. There are a couple of places where `CMakeLists` need to be changed when using `-DCMAKE_UNITY_BUILD=ON` with `-DBUILD_QJS_LIBC=ON` but I'll make a separate PR for that